vpd-tool/src/vpd_tool.cpp

```diff
@@ -0,0 +1,60 @@
+#include "vpd_tool.hpp"
+
+#include "constants.hpp"
+#include "types.hpp"
+#include "utils.hpp"
+
+#include <iostream>
+
+namespace vpd
+{
+void VpdTool::readKeyword(const std::string& i_fruPath,
+                          const std::string& i_recordName,
+                          const std::string& i_keywordName, bool i_onHardware,
+                          const std::string& i_fileToSave)
+{
+    types::DbusVariantType l_keywordValue;
+    if (i_onHardware)
+    {
+        std::cout << "Read from h/w" << std::endl;
+        // TODO: uncomment when implementation of the API goes in
+        /*l_keywordValue = utils::readKeywordFromHardware(
+            constants::vpdManagerService,
+           constants::vpdManagerObjectPath,
+            constants::vpdManagerIntfName, i_fruPath,
+            make_tuple(i_recordName, i_keywordName));*/
+    }
+    else
+    {
+        std::string l_inventoryObjectPath(constants::baseInventoryPath +
+                                          i_fruPath);
+
+        l_keywordValue = utils::readDbusProperty(
+            constants::inventoryManagerService, l_inventoryObjectPath,
+            constants::ipzVpdIntf + i_recordName, i_keywordName);
+    }
+
+    if (const auto l_value = std::get_if<types::BinaryVector>(&l_keywordValue))
+    {
+        const std::string& l_keywordStrValue =
+            utils::getPrintableValue(*l_value);
+
+        // TODO: handle in case result should be redirect to a file.
+        (void)i_fileToSave;
```

Comment:
Can you please this check 
if(i_fileToSave.empty()
{
// print in JSON
copy the lines 
}
else
{
// TODO: handle in case result should be redirect to a file.
}